nautobot/dcim/views.py

```diff
@@ -219,6 +219,7 @@ class LocationTypeUIViewSet(
     view_mixins.ObjectEditViewMixin,
     view_mixins.ObjectDestroyViewMixin,
     view_mixins.ObjectBulkDestroyViewMixin,
+    view_mixins.ObjectBulkUpdateViewMixin,
```

Comment:
With this addition, I think this can just become a simple `class LocationTypeUIViewSet(NautobotUIViewSet):` declaration.